LibreNMS/OS/Smartax.php

```diff
@@ -22,7 +22,6 @@
  * @copyright  2018 TheGreatDoc
  * @author     TheGreatDoc <doctoruve@gmail.com>
  */
-
```

Comment:
Why all the line removals?

Reply:
No reason, I reverted them. In future PR I want to migrate processors to yaml.